addons/longhorn/template/base/install.sh

```diff
@@ -20,10 +21,21 @@ function longhorn_pre_init() {
     fi
 
     # Can only upgrade 1 minor version at a time
-    longhorn_bail_upgrade_unsupported
+    if ! longhorn_can_upgrade ; then
+        if ! confirmY ; then
```

Comment:
I don't think this confirmy is very clear - should the user press y to continue, or to bail?
the message is:
```
        logWarn "Upgrades to Longhorn version ${LONGHORN_VERSION} from versions prior to 1.${previous_version_minor}.x are unsupported."
```